### PR TITLE
Update abbreviation list

### PR DESCRIPTION
**Pull request by yukimoritani**

### CODE REVIEW

File: misc/PFS_Abbreviations.tsv [hunk 1]
```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
...
 FCA	SpS	Subcomponent	Fiber Cable A Assembly	SpS assembly of fiber slit and connectors with Cable B.
 fcc	ICS	Actor	Field Center Camera	Software to control field center camera of PFI. ASIAA has a responsibility for its development.
 FEL	PFI	Subcomponent	Field Element	PFI component. In order to have the similar optical path to HSC, a glass plate of 54 mm thick is accommodated at the bottom of PFI.
 FEM	PFI	Subcomponent	Field Element Mounts	PFI component to mount the field element (FEL) to the Cobra bench (COB). CIT has a responsibility for its development.
 FFC	PFI	Subcomponent	Fixed Fiducial Camera	PFI component to take image of targets acquired on fixed fiducial fibers. ASIAA has a responsibility for its development.
 FFF	PFI	Subcomponent	Fixed Fiducial Fibers	Fibers fixed on the Cobra bench (COB) used as a reference of fiber position. There are 96 FFF in total, 84 for science fibers, and 12 for AG cameras.
+FFI	PFI	Subcomponent	Fiducial Fiber Illuminator	PFI component to back-illuminate fixed fiducial fibers and take image of the light coming through the fiducial fibers.
```
Comment (himorin) [inline misc/PFS_Abbreviations.tsv:55]: with adding this, need to remove FIS?

File: misc/PFS_Abbreviations.tsv [hunk 2]
```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
...
+FPRD	General	Document	Functional Performance Requirements Document	One of the documents at Preliminary Design Review. FPRD describes all requirements for PFS instrument to satisfy scientific requirements. For system-level requirements, the latest version is available on PBworks.
+fps	ICS	Actor	Fiber Positioning Sequencer	PFI software to operate fiber configuration. FPS controls MCS exposure, derives fibers positions from measured centroids, and sends them with command to MPS to move fibers. ASIAA has a responsibility for its development.
 FPSA	SpS	Subcomponent	Fiber Slit Positioning Sub-Assembly	SpS assembly to position fiber slit. FPSA consists of Hexapod to tilt and shift the Fiber Slit sub-Assembly (FSA).
-FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	FOCCos assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
+FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	Focus assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
```
Comment (himorin) [inline misc/PFS_Abbreviations.tsv]: focus or fiber slit? focus assembly seems to include hex-stage..

File: misc/PFS_Abbreviations.tsv [hunk 3]
```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
...
 ICS	Software	General	Instrument Control Software	Software packages to operate PFS instrument. Individual actuators, which control smaller components, exchange command, status with each other via messaging hub systems (MHS).
-IIS	SpS	Subcomponent	Internal Illumination Source	Light source ofSpS to illuminate engineering fibers.
-IPMU	General	Organization	Kavli Institute for the Physics and Mathematics of the Universe, the University of Tokyo	PFS collaborator (PI) taking initiative to proceed the project. IPMU also has a responsibility to develop softwares interfacing between PFS and Subaru, and execute system commissioning.
-IR3	Subaru	Facility	IR-side 3rd floor in the dome building 	The place on which the UPS, chillers for spectrgraph modules and the spectrograph clean room (SCR) will be accomodated.
-IR4	Subaru	Facility	IR-side 4th floor in the dome building 	The place on which the spectrgraph clean room (SCR) will be constructed.
+iic	ICS	Actor	Instrument Interface and Controller	ICS software module which organizes operation of PFS subsystems in an exposure sequence, in harmony with the telescope.
```
Comment (himorin) [inline misc/PFS_Abbreviations.tsv:74]: iic in lower or upper?

File: misc/PFS_Abbreviations.tsv [hunk 4]
```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
...
+IIS	SpS	Subcomponent	Internal Illumination Source	Light source of SpS to illuminate engineering fibers.
+IPMU	General	Organization	Kavli Institute for the Physics and Mathematics of the Universe, the University of Tokyo	PFS collaborator (PI) taking initiative to proceed the project. IPMU also has a responsibility to develop software interfacing between PFS and Subaru, and execute system commissioning.
+IR3	Subaru	Facility	IR-side 3rd floor in the dome building 	The place on which the UPS, chillers for spectrograph modules and the spectrograph clean room (SCR) will be accommodated.
+IR4	Subaru	Facility	IR-side 4th floor in the dome building 	The place on which the spectrograph clean room (SCR) will be constructed.
```
Comment (himorin) [inline misc/PFS_Abbreviations.tsv]: better to mention on TUE-IR also?

File: misc/PFS_Abbreviations.tsv [hunk 5]
```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
...
 MAC	ICS	General	MLP1 AG Converter	Software as an interface between PFS Auto Guide Cameras (AGCs) and Subaru Mid-Level Processors 1 (MLP1). MAC calculates tracking error using information from AGCs and send them to MLP1. MAC also sends AGCs image to guide viewer after converting proper format. IPMU has a responsibility for its development.
 MCPS	PFI	Subcomponent	Master Controller Power Supply	Device of Positioner Remote Electronics (PRE). MCPS supplies DC power to controller modules of Cobra positioners.
-MCS	PFS	Subsystem	Metrology Camera System	PFS instrument subsystem installed to the Cassegrain focus. MCS takes image of back-illuminated fibers, and measures their centroids. ASIAA has a resiponsibility for ites development.
+MCS	PFS	Subsystem	Metrology Camera System	PFS instrument subsystem installed to the Cassegrain focus. MCS takes image of back-illuminated fibers, and measures their centroids. ASIAA has a responsibility for its development.
+meb	ICS	Actor	MCS Electronic Box interface	Software to control power supply and telemetry system for MCS. ASIAA has a responsibility for its development.
```
Comment (himorin) [inline misc/PFS_Abbreviations.tsv]: also hardware definition exists??

File: misc/PFS_Abbreviations.tsv (hunk 3 above)
Comment (yukimoritani) [reply misc/PFS_Abbreviations.tsv:74]: Well... I don't know. but other actors are written in lower case.

File: misc/PFS_Abbreviations.tsv (hunk 2 above)
Comment (yukimoritani) [reply misc/PFS_Abbreviations.tsv]: Ah, thank you for pointing.
Recently SpS team discussed and FSA is now used for "Fiber Slit Sub-Assembly"
Hex staged one is called FPSA or FPA (Fiber Slit Positioning Sub-Assembly)
I should have modified taking these updates into account.

File: misc/PFS_Abbreviations.tsv (hunk 1 above)
Comment (yukimoritani) [reply misc/PFS_Abbreviations.tsv:55]: FFI is layer 3, and FIS is layer 4.
I tried to include abbreviations for down to layer 4, for PFI...

File: misc/PFS_Abbreviations.tsv (hunk 4 above)
Comment (yukimoritani) [reply misc/PFS_Abbreviations.tsv]: If someone (probably from Subaru) still call IR4 TUE-IR, I agree with you...
I'm not sure, so I'll add it. Thank you.

File: misc/PFS_Abbreviations.tsv (hunk 5 above)
Comment (yukimoritani) [reply misc/PFS_Abbreviations.tsv]: Thank you for suggestion. As I just followed product tree, I missed it. I'll add it.

**Review by himorin — APPROVED** [review]

File: misc/PFS_Abbreviations.tsv (hunk 3 above)
Comment (hassanxp) [reply misc/PFS_Abbreviations.tsv:74]: ~~I think IIC as an acronym should be in upper case.~~

Comment (hassanxp) [reply misc/PFS_Abbreviations.tsv:74]: Looking at the other actor abbreviations, perhaps it's better to leave it lower for consistency.

**Review by hassanxp — APPROVED** [review]
I think that the changes are all fine to be merged to master.

**Comment by yukimoritani** [comment]
I pass the review by @naoyukitamura to Hassan's work, which will be done after this ticked.

**Pull request merged**